x/participationrewards/keeper/msg_server_test.go

```diff
@@ -225,7 +225,7 @@ func (s *KeeperTestSuite) Test_msgServer_SubmitClaim() {
 				msg = types.MsgSubmitClaim{
 					UserAddress: address.String(),
 					Zone:        "cosmoshub-4",
-					SrcZone:     "testchain1",
+					SrcZone:     "testchain1-1", // wrong src_zone string maybe ?
```

Comment:
the two zones set up by setupZone are testchain1 and testchain2, unless this changed between v0.46 and v0.47

Reply:
ibc-go v7 use a new way to generate chainID so its now `testchain1-1`
ref: https://github.com/cosmos/ibc-go/blob/540d64bcc750064a6dccf2b52a14a438c859d644/testing/coordinator.go#L178